Web/ConnectionUtil.java

```diff
@@ -0,0 +1,24 @@
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionUtil {
+    private static final String DB_URI = "jdbc:mysql://localhost:3306/jspassignment1";
+    private static final String USER_NAME = "root";
+    private static final String PASSWORD = "mysql";
+
+    public static Connection createConnection(HttpServletResponse response) throws IOException {
```

Comment:
This class is only for making database connection. Here is no need of HttpServletResponse, if there will any error then you simply return the null to the calling servlet and calling servlet will response on the error.jsp. Please correct it.